dbt_platform_helper/commands/copilot.py

```diff
@@ -68,7 +70,7 @@ def copilot():
     check_platform_helper_version_needs_update()
 
 
-def _validate_and_normalise_config(config_file):
+def _validate_and_normalise_addons_config(config_file, key_in_config_file=None):
     """Load the addons.yaml file, validate it and return the normalised config
```

Comment:
```suggestion
    """Load the extensions.yaml file, validate it and return the normalised config
```

Reply:
Should this actually be talking about `platform-config.yml` now?